Makefile.am

```diff
@@ -69,6 +69,7 @@ endif
 if CPU_AARCH64
 libisal_la_SOURCES += ${lsrc_aarch64}
 other_tests += ${other_tests_aarch64}
+my_CFLAGS += -march=armv8-a+crc+crypto
```

Comment:
This is necessary, compiler needs this option to support AARCH64 special instructions.
It will not affect other arm machines, because 64 bit arm machines is supported from start armv8, all 64 bit arm machines should support the feature.

Reply:
Please ignore my previous comment, thanks for this concern, I will delete this line. arm assembly code no need this option.